src/binding.cc

```diff
@@ -77,10 +87,22 @@ void HandleException(v8::Local<v8::Context> context,
                                       message->GetScriptResourceName());
     v8::String::Utf8Value line_str(isolate, line);
     v8::String::Utf8Value col_str(isolate, column);
-    char buf[512];
     snprintf(buf, sizeof(buf), "%s\n%s %s:%s\n", ToCString(exceptionStr),
              ToCString(script_name), ToCString(line_str), ToCString(col_str));
-    d->last_exception = std::string(buf);
+    *exception_str += buf;
+  }
+}
+
+void HandleException(v8::Local<v8::Context> context,
+                     v8::Local<v8::Value> exception) {
+  v8::Isolate* isolate = context->GetIsolate();
+  Deno* d = FromIsolate(isolate);
+  std::string exception_str;
+  HandleExceptionStr(context, exception, &exception_str);
```

Comment:
Passing a pointer to std::string seems rather un-idiomatic. But whatever.

Reply:
Yeah - that's a programmer error - but I will fix it later to avoid having to rebase a bunch of branches :)